src/compiler/crystal/semantic/lib.cr

```diff
@@ -87,7 +87,7 @@ class Crystal::Call
     # (and we checked that all args are covered), and we
     # remove our named args.
     sorted_named_args.sort_by! &.[0]
-    sorted_named_args.each do |(index, named_arg)|
+    sorted_named_args.each do |(_, named_arg)|
```

Comment:
I think at lines like this it could just stay ` sorted_named_args.each do |(index, named_arg)| ` because I think it's better to have a proper name for it instead of replacing it by an underscore. Even if it's not used.

Reply:
Elixir follows the same convention which helps to reduce noise IMO.